### ISSUE TITLE
can we share video ?

### ISSUE THREAD

**Issue by ggsonic** [zhouteng0217/ShareExtend#4]
using image_picker,we can pickVideo from photo gallery,  how can we share the mp4 video file from the photo gallery?

**Comment by zhouteng0217** [comment]
you can share video with "file" parameter.
```
File f = await ImagePicker.pickVideo(source: ImageSource.gallery);
ShareExtend.share(f.path, "file");
```

**Comment by ggsonic** [comment]
it  didn't work properly, the mp4 video file finally not successfully shared. tested on my RedMi and Sony  Xperia android phones

**Comment by zhouteng0217** [comment]
> it didn't work properly, the mp4 video file finally not successfully shared. tested on my RedMi and Sony Xperia android phones

Did you choose the mp4 video file from other apps like Root Explorer?

**Comment by ggsonic** [comment]
I record video using my camera,then picked it  from photo gallery, then I Share it to WeChat, everything seems fine,but the video file was not really shared.you can see the error msg below. WeChat complained "This video is no longer available"

![111](https://user-images.githubusercontent.com/704873/54894190-65e4f000-4ef4-11e9-955d-57804cd637e7.jpg)

**Comment by zhouteng0217** [comment]
@ggsonic ,  I found that problem and later will publish a new version to fix it.

**Comment by zhouteng0217** [comment]
This issue has been fixed on the version 1.0.5,  please check it. Thanks for the issue report!
